cmake/modules/yaml-cpp.cmake

```diff
@@ -27,13 +27,18 @@ if(NOT USE_BUNDLED_DEPS)
 else()
   set(YAMLCPP_SRC "${PROJECT_BINARY_DIR}/yaml-cpp-prefix/src/yaml-cpp")
   message(STATUS "Using bundled yaml-cpp in '${YAMLCPP_SRC}'")
-  set(YAMLCPP_LIB "${YAMLCPP_SRC}/libyaml-cpp.a")
+  if(NOT WIN32)
+    set(YAMLCPP_LIB "${YAMLCPP_SRC}/libyaml-cpp.a")
+  else()
+    set(YAMLCPP_LIB "${YAMLCPP_SRC}/Debug/yaml-cpp.lib")
```

Comment:
I don't really know why it outputs to Debug, neither how to change this.
Of course `CMAKE_BUILD_TYPE` is not used in the project :/